src/users/containers/invite-user/invite-user.component.ts

```diff
@@ -142,7 +142,7 @@ export class InviteUserComponent implements OnInit, OnDestroy {
     return new FormGroup({
       firstName: this.createFormControl('', Validators.required),
       lastName: this.createFormControl('', Validators.required),
-      email: this.createFormControl('', [Validators.email, Validators.required]),
+      email: this.createFormControl('', [Validators.required, Validators.email]),
```

Comment:
How come these are flipped round?

Reply:
Because I wanted to put required first. nothing more!

Reply:
I tend to put mandatory checks ahead of field validation.